packages/core/src/index.ts

```diff
@@ -343,17 +346,19 @@ Signal.prototype._unsubscribe = function (node) {
 };
 
 Signal.prototype.subscribe = function (fn) {
-	return effect(() => {
-		const value = this.value;
-
-		const prevContext = evalContext;
-		evalContext = undefined;
-		try {
-			fn(value);
-		} finally {
-			evalContext = prevContext;
-		}
-	});
+	return effect(
+		() => {
+			const value = this.value;
+			const prevContext = evalContext;
+			evalContext = undefined;
+			try {
+				fn(value);
+			} finally {
+				evalContext = prevContext;
+			}
+		},
+		{ name: this.name + "-subscribe" }
```

Comment:
nit: could this just be `"subscribe"` and the debug lib calculates the name? the effect will only ever have a single dependency.